.github/workflows/docs.yml

```diff
@@ -29,6 +29,17 @@ jobs:
                     corepack enable
                     corepack prepare yarn@stable --activate
 
+            -   name: Set up Python
+                uses: actions/setup-python@v5
+                with:
+                    python-version: 3.8
```

Comment:
Why 3.8? Couldn't we use newer version?

Reply:
again, just a copy-paste from `apify-client` 

I have no idea what the latest version is (or what might break here), so I just went with it 😅